service/controller/key/machine_deployment.go

```diff
@@ -47,6 +47,16 @@ var (
 				WeightedCapacity: 1,
 			},
 		},
+		"m4.16xlarge": {
+			template.LaunchTemplateOverride{
+				InstanceType:     "m5.16xlarge",
```

Comment:
does the order matter? I looked some line above and see m4.4xlarge starts with m4.4xlarge and then m5.4xlarge

Reply:
I'd be surprised if it mattered. A few lines further below it's the other way around.